docs/install/uninstallEverest.md

```diff
@@ -1,61 +1,73 @@
 # Uninstall Everest
 
-No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including:
+You can run the commands below to remove all Everest resources including:
 
 - All Kubernetes objects created by Everest
-- All Docker containers created by Everest
-- All downloaded binaries and files like **everestctl**, **quickstart.yml**.
+- All downloaded binaries and files like **everestctl**.
 
 !!! note alert alert-primary "Warning"
-    Uninstalling Everest will remove all database clusters and associated data from the Kubernetes cluster!
+    Uninstalling Everest will remove all database clusters and associated data from the Kubernetes cluster, including backups!
 
     Before proceeding with Custom Resource Definitions (CRD) cleanup, check that any custom resources linked with CRDs are removed. This is a crucial step because deleting CRDs can potentially cause issues with any custom resources that depend on those definitions within the cluster.
 
 To uninstall Everest:
 {.power-number}
 
-!!! note alert alert-primary "Warning"
-    Follow these cleanup instructions after unprovisioning Everest via CLI.
+1. Uninstall Everest:
+
+    ```sh
+    everestctl uninstall
+    ```
 
-1. Remove ALL created database clusters:
+2. Remove ALL created database clusters:
 
     ```sh
     kubectl delete db --all -n percona-everest
     ```
 
-2. Remove ALL PVCs:
+3. Remove ALL backups:
+
+    ```sh
+    kubectl -n percona-everest delete job --all
+    kubectl -n percona-everest delete pxc-backup --all
+    kubectl -n percona-everest delete psmdb-backup --all
+    kubectl -n percona-everest delete pg-backup --all
+    kubectl -n percona-everest delete pod --all     
+    ```
+
+4. Remove ALL PVCs:
 
     ```sh
     kubectl delete pvc --all -n percona-everest
     ```
 
-3. List CSVs and remove those with a "*percona*" and "*everest*" prefix, or remove ALL operators:
+5. List CSVs and remove those with a "*percona*" and "*everest*" prefix, or remove ALL operators:
 
     ```sh
     kubectl delete sub --all -n percona-everest
     kubectl delete ip --all -n percona-everest
     kubectl delete csv --all -n percona-everest
     ```
 
-4. List ALL CRDs that will be removed:
+6. List ALL CRDs that will be removed:
```

Comment:
```suggestion
6. List ALL CRDs that will be removed using the following command. 
Keep in mind that this command will exclusively list CRDs linked to Everest and skip any CRDs associated with standalone Percona Operators. This distinction is crucial because if there are instances of standalone Percona Operators existing within any namespace in your Kubernetes environment, you should avoid inadvertently deleting their CRDs.
```